tuner/tuner/libtuner_test.py

```diff
@@ -108,12 +108,15 @@ def test_parse_dispatch_benchmark_results():
     spec_dir = base_path / "specs"
     path_config = libtuner.PathConfig()
     object.__setattr__(path_config, "specs_dir", spec_dir)
+    object.__setattr__(path_config, "output_json", "output.json")
 
     mock_result_1 = MagicMock()
-    mock_result_1.run_result.process_res.stdout = "process_time/real_time_mean 100.0 us"
+    mock_json_1 = {"benchmarks": [{"real_time": 100.0}]}
+    mock_result_1.run_result.process_res.stdout = json.dumps(mock_json_1)
     mock_result_1.candidate_id = 1
     mock_result_2 = MagicMock()
-    mock_result_2.run_result.process_res.stdout = "process_time/real_time_mean 200.0 us"
+    mock_json_2 = {"benchmarks": [{"real_time": 200.0}]}
```

Comment:
We should also have a test cast with a smaller number like: `"real_time": 3.4591828516259519e-02`